src/routes.js

```diff
@@ -0,0 +1,21 @@
+import Home from './Home'
+import Play from './Play'
+import playGame from './playGame'
+
+export const routes = [
+   {
+      path: '/',
+      component : Home,
```

Comment:
How are we planning to have this change to be the dashboard/game list once you are logged in, or are we always going to have this page?

Reply:
An open question. I'm actually having some issues getting the http get from the ajax call to work as I want to after the redirect.

Reply:
Alright cool we can let the auth stuff be open till the next PR